Sources/TSCBasic/Process/Process.swift

```diff
@@ -239,45 +239,7 @@ public final class Process {
         }
     }
 
-    // deprecated 2/2022, remove once client migrate to logging handler
-    @available(*, deprecated)
-    public static var verbose: Bool {
-        get {
-            Self.loggingHandler != nil
-        } set {
-            Self.loggingHandler = newValue ? Self.logToStdout: .none
-        }
-    }
-
-    private var _loggingHandler: LoggingHandler?
-
-    // the log and setter are only required to backward support verbose setter.
-    // remove and make loggingHandler a let property once verbose is deprecated
-    private let loggingHandlerLock = NSLock()
-    public private(set) var loggingHandler: LoggingHandler? {
-        get {
-            self.loggingHandlerLock.withLock {
-                self._loggingHandler
-            }
-        }
-        set {
-            self.loggingHandlerLock.withLock {
-                self._loggingHandler = newValue
-            }
-        }
-    }
-
-    // deprecated 2/2022, remove once client migrate to logging handler
-    // also simplify loggingHandler (see above) once this is removed
-    @available(*, deprecated)
-    public var verbose: Bool {
-        get {
-            self.loggingHandler != nil
-        }
-        set {
-            self.loggingHandler = newValue ? Self.logToStdout : .none
-        }
-    }
+    public let loggingHandler: LoggingHandler?
```

Comment:
Without `private(set)` this will fail to build due to the `static` version of `loggingHandler`. Let's deprecate the static one while we're at it.

```suggestion
    public private(set) let loggingHandler: LoggingHandler?
```

Reply:
Odd, it seems to be building just fine for me locally.

Reply:
Ah, you're right, the static variable uses a separate static underscored property...